Source/Core/DolphinQt2/Config/Mapping/Hotkey3D.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
I have an idea but I would like to "outsource" it to another PR to apply this evenly (Saving even more LOC!).